packages/office-ui-fabric-react/src/components/Icon/Icon.scss

```diff
@@ -0,0 +1,12 @@
+
+.ms-Icon-ImageContainer {
+  overflow: hidden
+}
+
+.ms-Icon-ImageContainer.ms-ContextualMenu-icon {
+  margin-top: 10px
```

Comment:
Why does this need margin top? And why is .ms-ContextualMenuIcon in the icon class?